Change C# namespace to Milvus.Client

roji opened the pull request:
As per offline discussions, the Milvus .NET SDK will have the nuget name and namespace Milvus.Client, which is the idiomatic .NET naming (rather than IO.Milvus).

/cc @weianweigan @stephentoub @tawalke

issue: #184

weianweigan commented:
/assign @yhmo

weianweigan commented:
@roji [common.proto](https://github.com/milvus-io/milvus-proto/blob/master/proto/common.proto) and [schema.proto](https://github.com/milvus-io/milvus-proto/blob/master/proto/schema.proto) also have a C# namespace. Maybe we should change them together ?
It seems that the active branch for **[milvus-proto](https://github.com/milvus-io/milvus-proto)** is in version 2.2. Perhaps the target branch should be set to [branch 2.2](https://github.com/milvus-io/milvus-proto/tree/2.2) ?

weianweigan commented:
lgtm

roji commented:
Thanks @weianweigan! I changed the C# namespace in the other proto files as well and rebased the PR against the 2.2 branch. I can also see commits in master, so it's a bit confusing - let me know if all is good.

yhmo commented:
/lgtm
/approve
